### ISSUE TITLE
Add possibility to revert the override

### ISSUE THREAD

**Issue by teamplayer3** [eopb/cargo-override#130]
I think the title describes it well. It would be cool to revert an override. This could be done by hand, by removing the line, but it would standardize the process of handling the overrides.

## Implementation

```
cargo override --revert <package-name>
```

**Comment by eopb** [comment]
I agree this would be great to have. I'd probably go for `cargo rm-override -p <pkgid>` rather than a `--revert` flag to closer match what cargo already has for `cargo add` and `cargo remove`.

This shouldn't be too hard to implement so I'll label this issue "good first issue"

**Comment by teamplayer3** [comment]
I will start working on it.

**Comment by eopb** [comment]
Awesome! Let me know if you run into any issues
